src/codegen.cpp

```diff
@@ -4748,15 +4747,6 @@ static void init_julia_llvm_env(Module *m)
                          "jl_alloc_tuple", m);
     add_named_global(jl_alloc_tuple_func, (void*)&jl_alloc_tuple);
 
-    std::vector<Type *> puts_args(0);
-    puts_args.push_back(T_pint8);
-    puts_args.push_back(T_pint8);
-    jlputs_func =
-        Function::Create(FunctionType::get(T_void, puts_args, false),
-                         Function::ExternalLinkage,
-                         "jl_puts", m);
-    add_named_global(jlputs_func, (void*)&jl_puts);
-
```

Comment:
the `jlstderr_var` could go too. i haven't needed to JIT a call to `jl_puts(jlstderr_var)` in quite some time. the addition of `code_llvm` was probably what mostly replaced that need.